changelogs/head.asciidoc

```diff
@@ -70,3 +70,7 @@ https://github.com/elastic/apm-server/compare/7.15\...master[View commits]
 - Add deprecation warning for `sampling.keep_unsampled=true` {pull}6285[6285]
 - `processors.*` config, which was never officially supported in apm-server, is now explicitly deprecated and will be removed in 8.0 {pull}6367[6367]
 - Support for uploading source maps to APM Server is deprecated, and will be removed in 8.0. Users should use the new Kibana REST API in conjunction with the integration package {pull}6432[6432]
+
+[float]
+==== Licensing Changes
+- Updated the `x-pack` source files license to the Elastic License 2.0 {pull}6524[6524]
```

Comment:
@bmorelli25 Happy to hear your thoughts on the matter, we discussed it with @simitt briefly on Slack but do let me know if we should rename or change the header's name.

Reply:
👍 LGTM